docs/troubleshoot/app-framework/zis-error-codes.md

```diff
@@ -0,0 +1,1985 @@
+support if needed.
+
+Possible return codes:
+
+| _return_code_                      | Action                                        |
+|------------------------------------|-----------------------------------------------|
+| RC_CMS_GLOBAL_AREA_NULL(12)        | Ignore if you have not run this ZIS after IPL |
+| RC_CMS_ZVT_NULL(47)                | Ignore if you have not run any ZIS after IPL  |
+| RC_CMS_ZVTE_CHAIN_LOOP(66)         | Contact support                               |
+| RC_CMS_ZVTE_CHAIN_NOT_LOCKED(67)   | Contact support                               |
+| RC_CMS_ZVTE_CHAIN_NOT_RELEASED(68) | Contact support                               |
+
+### ZWES0109I
+
+Core server ready
+
+**Reason:**
+
+The cross-memory server initialized and it is ready to accept program calls.
+
+**Action:**
+
+No action required.
+
+### ZWES0110E
+
+Main loop unexpectedly terminated
+
+**Reason:**
+
+The cross-memory server detected an incorrect state in the main loop.
+
+**Action:**
+
+The cross-memory server terminates. Contact support.
+
+### ZWES0111I
+
+Main loop terminated
+
+**Reason:**
+
+The main loop of this cross-memory server successfully terminated upon shutdown.
+
+**Action:**
+
+No action required.
+
+### ZWES0112E
+
+Termination step '_step_name_' failed, RC = _return_code_
+
+**Reason:**
+
+A cross-memory server's termination step failed.
+
+**Action:**
+
+The termination process continues. Review the messages preceding this message.
+If you cannot resolve the issue, contact support.
+
+### ZWES0113I
+
+Termination step '_step_name_' successfully completed
+
+**Reason:**
+
+A cross-memory server's termination step completed successfully.
+
+**Action:**
+
+No action required.
+
+### ZWES0114I
+
+Core server stopped
+
+**Reason:**
+
+The cross-memory server successfully stopped.
+
+**Action:**
+
+No action required.
+
+### ZWES0115E
+
+Core server stopped with an error, status = _status_code_
+
+**Reason:**
+
+The cross-memory server stopped with a non-zero status.
+
+**Action:**
+
+Review the messages preceding this message. Contact support if you cannot
+resolve the issue.
+
+### ZWES0116E
+
+Core server is abnormally terminating
+
+**Reason:**
+
+An abend occurred in this cross-memory server.
+
+**Action:**
+
+Review any messages and errors preceding this message and contact support if you
+cannot resolve the issue.
+
+### ZWES0117E
+
+Not APF-authorized (_testauth_status_)
+
+**Reason:**
+
+One or more data sets in the STEPLIB concatenation is not APF-authorized.
+
+**Action:**
+
+The cross-memory server terminates. Ensure that all the STEPLIB data sets are
+APF-authorized.
+
+### ZWES0118E
+
+Core server started in wrong key _key_
+
+**Reason:**
+
+The cross-memory server detected that it was running in the wrong key.
+
+**Action:**
+
+The cross-memory server terminates. Ensure that you have added the correct
+PPT-entry (
+see [the documentation](https://docs.zowe.org/stable/user-guide/configure-xmem-server/))
```

Comment:
These types of URLs could break if someone changes the doc tech.
Instead, try relative:
(../../user-guide/configure-xmem-server.md)

you will see examples of this elsewhere, https://github.com/ifakhrutdinov/docs-site/blob/feature/zis-message-ids/docs/user-guide/systemrequirements-zos.md?plain=1#L99

Reply:
Fixed.